corehq/apps/app_manager/templates/app_manager/partials/form_designer_requirejs_config.html

```diff
@@ -5,6 +5,7 @@
 define("jquery", [], function () { return window.jQuery; });
 define("jquery.bootstrap", ["jquery"], function () {});
 define("underscore", [], function () { return window._; });
+define("moment", [], function () { return window.moment; });
```

Comment:
I was running into some issues with moment and requirejs. Not sure if this was the right way to do this but it worked. @orangejenny do you own this?